hutch_python/epics_arch.py

```diff
@@ -69,75 +91,20 @@ def epics_arch_qs(args):
 
     >>> epicsarch-qs xpplv6818 --dry-run
     """
-    args = parser.parse_args(args)
-
-    if args.experiment and not args.dry_run:
+    file_path = None
+    if experiment and not dry_run:
         # set the path to write the epicsArch file to.
-        if args.path:
-            overwrite_path(args.path)
-        elif args.hutch:
-            overwrite_hutch(args.hutch)
-        else:
-            set_path(args.experiment)
-        create_file(args.experiment)
-    elif args.hutch:
-        overwrite_hutch(args.hutch)
-    elif args.path:
-        overwrite_path(args.path)
-    elif args.dry_run:
-        if args.experiment:
-            print_dry_run(args.experiment)
+        if path:
```

Comment:
I am still not sure if I'm thinking correctly here about the path.

Reply:
I think it's looking better 👍